### ISSUE TITLE
UI content is shifted to the left by 3 pixels

### ISSUE THREAD

**Issue by barbaragomes** [GeekyAnts/NativeBase#679]
Hi there,

I have noticed a weird behavior on the content of native-base containers when using the native-base Drawer.
All the views inside the Content are shifted to the left by 3 pixels.

If you inspect the view, you can see that the padding left is -3

## react-native, react and native-base version
react-native: 0.41.0
native-base: 2.0.9

## Expected behaviour

No padding added

## Actual behaviour

View is shifted left by 3 pixels

## Steps to reproduce (code snippet or screenshot)

```html
<Container>
    <Image source={backgroundImage} style={styles.backgroundImg}>

        <Header style={styles.header}>
            <Left>
                <Button transparent onPress={() => Actions.login()}>
                    <Icon name='long-arrow-left' style={{color: '#fff' }} />
                </Button>
            </Left>
            <Body>
                <Title style={styles.headerText}>Sign Up</Title>
            </Body>
            <Right></Right>
        </Header>

        <Content scrollEnabled={false} style={styles.container}>
            <View style={styles.top}>
            </View>
            <Text style={styles.mainTxt}>Sign-Up</Text>

            <View style={styles.main}>
                <Label style={styles.label}>Full Name</Label>

                <Item style={styles.input}>
                    <Icon name='user-o' style={styles.icon}/>
                    <Input style={styles.inputText} placeholder='Jim Bim' />
                </Item>
                <Label style={styles.label}>Email</Label>

                <Item style={styles.input}>
                    <Icon name='at' style={styles.icon}/>
                    <Input style={styles.inputText} placeholder='your@email.com' />
                </Item>
                <Label style={styles.label}>Password</Label>

                <Item  style={styles.input}>
                    <Icon name='lock' style={styles.icon}/>
                    <Input style={styles.inputText}  secureTextEntry={true} placeholder='Password'/>
                </Item>
                <Button style={styles.signUp} block rounded onPress={() => Actions.signUp()}>
                    <Text style={styles.buttonText}>Free Sign Up</Text>
                </Button>
                <Text style={styles.legalText}>TEXT<Text style={styles.links}>Terms of Service</Text> and <Text style={styles.links}>Privacy Policy</Text>.</Text>
            </View>

        </Content>
    </Image>
</Container>
```

## Screenshot of emulator/device
![padding](https://cloud.githubusercontent.com/assets/13023284/24265802/fdb651d6-0fc1-11e7-942c-0071060104f8.png)

![content](https://cloud.githubusercontent.com/assets/13023284/24265848/1c7635f0-0fc2-11e7-8949-1afec63e162a.png)

## Is the bug present in both ios and android or in any one of them?

We only use iOS

## Any other additional info which would help us debug the issue quicker.
If we remove the drawer from root, then the shift goes away

**Comment by krishkal** [comment]
This is solved by adding 
`closedDrawerOffset={0}`
to the <Drawer> component.

Apparently, whoever implemented the Drawer component within NativeBase used the Example code from [https://github.com/root-two/react-native-drawer#usage](url) which has `closedDrawerOffset={-3}`. It would be nice if NativeBase left it at the default 0.

**Comment by sankhadeeproy007** [comment]
Fixed with 2.1.0
